frontend/src/App.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// 테스트용 스타일링된 컴포넌트 생성
+const Button = styled.button`
```

Comment:
style.js 같은 파일을 만들어서 스타일링을 한 곳에 모으고 import하면 더 좋을 것 같습니다!

```suggestion

import * as S from styled.js;

function App() {
  return (
    <>
       <S.Button>클릭하세요</S.Button>
    </>
  );
}
```